src/main/java/uk/gov/ons/ssdc/caseprocessor/service/CaseService.java

```diff
@@ -58,12 +59,20 @@ private EventDTO prepareCaseEvent(Case caze, EventHeaderDTO eventHeader) {
     caseUpdate.setCollectionExerciseId(caze.getCollectionExercise().getId());
     caseUpdate.setSurveyId(caze.getCollectionExercise().getSurvey().getId());
     caseUpdate.setSample(caze.getSample());
+
     caseUpdate.setInvalid(caze.isInvalid());
     if (caze.getRefusalReceived() != null) {
       caseUpdate.setRefusalReceived(RefusalTypeDTO.valueOf(caze.getRefusalReceived().name()));
     } else {
       caseUpdate.setRefusalReceived(null);
     }
+    caseUpdate.setSampleSensitive(caze.getSampleSensitive());
+
+    // Due to time taken only run this is there's sampleSensitive data on the case
```

Comment:
Time taken? Likely to be sub millisecond. More like that it's a _CPU expensive_ operation, not a particularly time consuming one.